packages/rancher-monitoring/generated-changes/overlay/files/rancher/etcd/etcd-metrics-detail.json

```diff
@@ -63,13 +63,13 @@
         "steppedLine": false,
         "targets": [
           {
-            "expr": "sum(rate(etcd_network_client_grpc_received_bytes_total{job=\"kube-etcd\"}[5m])) by (instance)",
+            "expr": "sum(rate(etcd_network_client_grpc_received_bytes_total{job=\"kube-etcd\"}[$__rate_interval])) by (instance)",
```

Comment:
Guarantees that all the granularity will be at 4 times whatever Prometheus's scrape interval is.